react/game/src/app/components/Form/Button/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styles from './styles.scss';
+
+function Button({ type, disabled, title, children, onClick, classes, visible }) {
+  return visible ? (
+    <button
+      className={`${styles.appButton} ${classes}`}
+      type={type}
+      disabled={disabled}
+      title={title}
+      onClick={onClick}
+    >
+      {children}
+    </button>
+  ) : null;
+}
+
+Button.propTypes = {
+  title: PropTypes.string,
```

Comment:
`isRequired`

Reply:
Wally, the `title`, `placeholder` and `id` should not be required, because these properties of tag elements are just optional: the title just shows to the user a hint when hovering, the placeholder and id properties are not absolutely necessary or obligatory to make a Form work in the proper way. 
So about the `onClick` method, if we have a button of `type="submit"` the onClick method is not necessary... so that is why i just put it optional.